### PR TITLE
Incremental water level

### PR DESCRIPTION
**Pull request by JeffThorslund**
Went from a string based river level ("high", "medium", "low") to a range (in the format of [min,max] kept in state. Every hydraulic now has a range( [min,max] ) prop that indicates when that feature is "in". The hydraulics render based on if their range (when the feature is "in") falls within the the state range.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 .App {
   font-family: 'Roboto', sans-serif;
 }
+
+#gauge{
+  position: absolute;
+  right: 10vh;
+  top: 10vh;
+}
```
Comment (scott-force) [inline src/App.css:9]: files should always end with one blank newline

File: src/App.css [hunk 2]
```diff
@@ -1,3 +1,9 @@
 .App {
   font-family: 'Roboto', sans-serif;
 }
+
+#gauge{
```
Comment (scott-force) [inline src/App.css:5]: space before the bracket

File: src/App.css (hunk 1 above)
Comment (JeffThorslund) [reply src/App.css:9]: Thanks, Ill keep that in mind going forward.

File: src/App.css (hunk 2 above)
Comment (JeffThorslund) [reply src/App.css:5]: Sounds good.

**Pull request merged**